framework/tlvf/tlvf.py

```diff
@@ -851,10 +851,11 @@ def addClassParam(self, obj_meta, param_name, param_type, param_type_info, param
             if (param_val_const or param_val):
                 if is_int_len or is_const_len:
```

Comment:
Reminder: in the other PR, I made the (nitpick) comment that these two conditions could be swapped and the is_int/const_len can be merged with the previous one, which allows to have the m_parse__ condition only once in the generated C code.

Reply:
Done